### DISCUSSION TITLE
Feature Request: Clone a transaction

### DISCUSSION THREAD

**Discussion by imlonghao** [victorbalssa/abacus#82] [Ideas]
Usually, most of fields of my daily transaction are same for example the accounts and category, only the amount would be a little different.

It would be easier to add new transaction, I only need to clone an existing transaction, then modify the amount and save.
Instead of typing the same fields everyday.

The clone button/API is existed on the FireFly III web interface, but not in their open API, it may take times to implement it.

Hope to the clone button this in transaction editing page like this.

[RESET]
+ [CLONE]
[SUBMIT]

**Comment by victorbalssa** [comment]
@imlonghao done with hot fix 0.6.2.
Long press on transaction list item will duplicate current transaction.

**Reply by imlonghao** [reply]
Awesome, that should save me lots of time.

The way long press seems tricky, I wouldn't know that if you don't tell me.

**Reply by victorbalssa** [reply]
👍🏻 I will add a new slide right button for this, opposite to the delete button.

Thank you for sponsoring Abacus 🎉
